spec/factories/task.rb

```diff
@@ -132,6 +130,33 @@
         }
       end
     end
+    trait(:with_meta_data) do
+      meta_data do
+        {
+          'meta-data' => {
+            '@xmlns' => {'namespace' => 'custom_namespace.org'},
+            'namespace:meta' => {
+              '$1' => 'data',
+               '@xmlns' => {'namespace' => 'custom_namespace.org'},
```

Comment:
Yes this is redundant but seems to match the [specs](http://www.sklar.com/badgerfish/) (Example 9)

Reply:
Okay, I see. Probably, we've had this discussion before, I think, but I don't remember the details. 

Of course, the example you linked performs the transformation as you'd suggested, specifying the XML namespace multiple times. However, this seems to be a limitation, as identified by the second point that should be tackled (list "What's broken and/or left to do?", same [source](http://www.sklar.com/badgerfish/)). Further, some [other converters](http://dropbox.ashlock.us/open311/json-xml/) handle those cases correctly and do not replicate the XML namespace.

Probably, we can discuss that once again. I would be in favor of avoiding the namespace redundancy, but this alone shouldn't be the only issue preventing this MR from being merged and used. Nevertheless, it might be a good follow-up task to tackle afterward (which should still be compatible to the code that is generated with the current implementation).

Reply:
I agree to reduce redundancy. Since I don't believe it to be a complex problem, I would like to fix it right away.

Reply:
Awesome, I like that! Yes, let's make the small change for our Dachsfisch gem and then use it here directly.

Reply:
done :)

Reply:
Perfect, thank you!